pkg/pgmodel/migrate.go

```diff
@@ -52,6 +52,7 @@ var (
 			"tracing-functions.sql",
 			"tracing-views.sql",
 			"remote-commands.sql",
+			"telemetry.sql",
```

Comment:
`remote-commands.sql` needs to be next to last. Would you move it to just above `apply_permissions.sql` and add a comment to that effect?